ui/src/user/components/SignUpPage.tsx

```diff
@@ -1,18 +1,31 @@
 import React from 'react';
 import { Row, Col, Card, Alert } from 'antd';
-import PropTypes from 'prop-types';
 
 import SingUpForm from './SingUpForm';
 import DocumentHead from '../../common/components/DocumentHead';
+import { FormikHelpers } from 'formik';
 
-function SignUpPage({ onSubmit, loading, error }) {
+function SignUpPage({
+  onSubmit,
+  loading,
+  error,
+}: {
+  onSubmit: ((
+    values: {},
+    formikHelpers: FormikHelpers<{}>
+  ) => void | Promise<any>) &
+    Function;
+  loading: boolean;
+  error: Error;
+}) {
   return (
     <>
       <DocumentHead title="Sign up" />
-      <Row className="h-100" type="flex" justify="center" align="middle">
+      <Row className="h-100" justify="center" align="middle">
```

Comment:
why are we removing `flex`?

Reply:
[Antd removed it from <Row /> component](https://ant.design/components/grid/#Row) and flex is now it's default property, TS recognizes this and highlights type="flex" as error, that's why I have to remove it.